modules/_labs/header/react/spec/GlobalHeader.spec.tsx

```diff
@@ -83,10 +83,12 @@ describe('GlobalHeader', () => {
         onMenuClick: jest.fn(),
         leftSlot: <SearchBar onSubmit={jest.fn()} />,
         isCollapsed: true,
+        themeColor: HeaderTheme.White,
```

Comment:
??

Reply:
the test fails when I don't add this as props which is weird because GlobalHeader renders a header which has this prop as a default.  Any reason why this must be added?